dom.bs

```diff
@@ -906,8 +906,9 @@ correct defaults.</p>
  <li><p>Set <var>event</var>'s <a>initialized flag</a>.
 
  <li>
-  <p>Initialize <var>event</var>'s {{Event/timeStamp}} attribute to a {{DOMHighResTimeStamp}}
-  representing the high resolution time from the <a>time origin</a> to <var>time</var>.
+  <p>Initialize <var>event</var>'s {{Event/timeStamp}} attribute to the
+  <a>relative high resolution coarse time</a> given <var>time</var> and <var>realm</var>'s
+  <a for=Realm>global object</a>.
```

Comment:
_realm_ might be null. Probably the easiest way to handle this is to use _event_'s relevant global object, deferring the null-handling to that location.

Reply:
Thanks, done